tests/integration/test_patients_listing.py

```
@@ -0,0 +1,121 @@
+"""Integration tests for GET /patients — US-BE-08.
+
+Seed triagem data (admissions 1–5):
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)